infrastructure/azure.tf

```diff
@@ -1239,34 +1181,15 @@ resource "azurerm_virtual_network_peering" "aks_cluster_to_redis" {
   allow_virtual_network_access = "true"
 }
 
-# Old Azure Container Service (Kubernetes) - start
+# Azure Container Service (Kubernetes) - start
 
 locals {
   # The ssh public key for the admin account on the k8s nodes is read from the
   # file stored in the "files" directory and named after the value of the
   # variable "azurerm_kubernetes_admin_ssh_publickey_file"
-  azurerm_kubernetes_admin_ssh_publickey = "${file("${path.module}/files/${var.azurerm_kubernetes_admin_ssh_publickey_file}")}"
+  azurerm_kubernetes_admin_ssh_publickey = "${"${path.module}/files/${var.azurerm_kubernetes_admin_ssh_publickey_file}"}"
```

Comment:
out of curiosity, what's the difference between the prev and proposed syntax?

Reply:
It's exactly the same syntax. I've just moved the file to somewhere else in order to be able to reuse the local variable twice instead of using it only once and have to write the same things as hard-coded value in another place